kaedama.py

```diff
@@ -82,9 +80,9 @@ def sanity_checks( params, inputq ):
     #
 
     # Name should be of the form DST_NAME_runXauau
```

Comment:
Some care needs to be exercised here.  This disables the naming convention check.